src/font/fontlibrary.c

```diff
@@ -896,6 +894,11 @@ void LCUI_InitFontLibrary( void )
 			break;
 		}
 	}
+#ifndef LCUI_BUILD_IN_WIN32
+	for( i = 0; i < MAX_FONTFILE_NUM; ++i ) {
+		free( (void*)(fonts[i].path) );

```

Comment:
You do not need to convert pointer type.

``` diff
- free( (void*)(fonts[i].path) );
+ free( fonts[i].path );
```

Reply:
Not converting the pointer type results in a compiler warning.
```diff
fontlibrary.c: In function ‘LCUI_InitFontLibrary’:
fontlibrary.c:898:9: warning: passing argument 1 of ‘free’ discards ‘const’ qualifier from pointer target type [-Wdiscarded-qualifiers]
>   free( fonts[i].path );
>         ^~~~~
In file included from fontlibrary.c:42:0:
/usr/include/stdlib.h:448:13: note: expected ‘void *’ but argument is of type ‘const char *’
> extern void free (void *__ptr) __THROW;
>             ^~~~
```